server/models/restaurant.js

```diff
@@ -17,4 +18,7 @@ const restaurantSchema = Schema({
   comment: [commentSchema],
 });
 
-module.exports = mongoose.model("Restaurant", restaurantSchema);
+module.exports = {
```

Comment:
U can separate Restaurants and Comments model into two files if u want

Reply:
sure

Reply:
dou hai suen, only one comment route seems weird

Reply:
i tried all with postman, so I merge ok?